### ISSUE TITLE
[feature request] Grab bot outfits

### ISSUE THREAD

**Issue by phron3sis** [Bot-detector/bot-detector#24]
More data to look at:
What is available to RuneLite that would be useful for analyzing bot farms? Here's some quick-starter ideas:
- Player name layout (alphnumeric, phonetic, etc...)
- Player clothes/weapons
- Player stats trained
- Player stats not trained
- etc.

**Comment by extreme4all** [comment]
hey thanks for creating an issue, we are currently using player stats (from hiscores) & player name + location data from the plugin.
we have thought about collecting clothers/ weapons, but we are pretty sure that this will mostly pollute the data set.

- to many unique combinations of clothes & weapons, 
- not all bots have the same clothes & weapons, 
- a bot creator could easily change or randomize his clothes & weapons

Thanks for the feedback, we are open for any suggestions, hypothesis on how we can improve our detection, plugin & user experience

**Comment by Ferrariic** [comment]
Thank you for your comment. We'll consider this for later after verifying that our model works well - but at this time it may be problematic to include for the reasons listed above by @extreme4all.

**Comment by extreme4all** [comment]
keeping open, to avoid duplicates

**Comment by phron3sis** [comment]
what if you gather the data and just store it? You don't have to feed it to your learning algo.

It could be a per-account type of "flag". in the future you could use it to help with predictions.

At the very minimum, detecting wooden shield and bronze sword would be IDEAL. I see hundred of combat lvl 70+ accounts still using tutorial island gear! They're obviously bots.

TL;DR
Gather data about players, don't feed it to your ML. let a human look at the data you gather :)

If you say no to me again, I'll get pictures of 10-20 bots killing giant frogs ALL wearing same outfit and wooden shield, broze sword...

**Comment by Ferrariic** [comment]
Ah I get what you are saying because I do see a lot of bots wearing the same gear - but at this stage we can't make use of that data for the following reasons:

- Adding 11 columns to ~700K players + a gradual increase per week, at this stage, would put a strain on our system's database in terms of query speed and size. 

- We may have to categorize every gearset out there to simplify it for the model, while possible through the wiki, may not be as beneficial at this stage as getting more hiscore data and location data from Jagex - and improving the parameters of our models for the ensemble. 

- The plugin is not equipped to handle gear info at this time. 

-  The models are not equipped to handle gear info as well, at this moment. We would like to try and make use of what data we can in our system. 

I like the idea, but we don't have the capacities to make it work at this moment - I also had this idea earlier on. These things come in steps. You have to build a foundation before you can start adding walls and windows.

**Comment by Ferrariic** [comment]
I'll keep this equipment idea up as we will consider it for the future, once we have built a good foundation for the project :)

**Comment by Ferrariic** [comment]
We can look into it :)

**Comment by Cyborger1** [comment]
Implemented in #88

**Comment by phron3sis** [comment]
> 
> 
> Implemented in #88

Badass!
